### DISCUSSION TITLE
Esp32 a2dp sink to i2s with aptx decoder

### DISCUSSION THREAD

**Discussion by simple-ee** [pschatzmann/arduino-audio-tools#666] [Q&A] [answered]
I'm wondering if anyone can help with trying to get a sketch working to decide aptx from a2dp sink.

The examples are a great help but trying to piece each part into a single sketch is where I get confused. I'm new to classes and the esp32 platform so any help would be greatly appreciated.

**Comment by pschatzmann** [comment, marked as answer]
I think you can consider this only if you really know what you are doing. 
This is even over my head, because you can't do this in Arduino using the standard ESP32 IDF...

**Reply by simple-ee** [reply]
Oh ok I can see the complexity just with sbc but thank you for the reply and clarification

**Reply by simple-ee** [reply]
Also one more question. 
Using your audio tools instead of the esp32 a2dp, how would you use the i2s stream to a2dp sink from a mobile device and then output via i2s? 
I see your example in the stream folder but that's for i2s input to a2dp out.
Thank you
